### DISCUSSION TITLE
Github student developer pack problem

### DISCUSSION THREAD

**Discussion by Nis-nischith** [community/community#106574] [GitHub Education]
### Select Topic Area

Question

### Body

Hi, ***! You were last verified as a null on Jan. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits.
It is showing approved but this message is displayed. It's been over 3 weeks and I haven't received the benefits yet. Please help.

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @Nis-nischith , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Comment by samus-aran** [comment]
Hi @Nis-nischith 👋 

Thank you for bringing this to our attention. We've passed this along to our internal team to investigate and when we have any updates we'll update this thread. 👍

**Reply by Nis-nischith** [reply]
Thanks a lot!

**Comment by samus-aran** [comment]
Hi @Nis-nischith That should be your status updated. You should've received the confirmation email.

- Connect with a local Campus Expert. For more information on campus experts, see "[About Campus Experts](https://education.github.com/students/experts)."
- Explore and claim offers for free industry tools from the [Student Developer Pack](https://education.github.com/pack).
- See upcoming in-person and virtual events for students, curated by GitHub Education and student leaders.
- View assignments from [GitHub Classroom](https://classroom.github.com/) with upcoming due dates.
- Stay in the know with [Campus TV](https://www.twitch.tv/githubeducation) episodes. Campus TV is created by GitHub and student community leaders and can be watched live or on demand.
- Discover and contribute to student-created repositories from GitHub Community Exchange. For more information, see "[About GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange)."

It's great to have you in the GitHub Education community and we hope to see you around 👋

**Reply by Nis-nischith** [reply]
Yes I received the benefits!
Thnx a lot 💓

**Comment by Grimthorr** [comment]
@samus-aran I have the same issue with my teacher benefits - the status is `Approved` but I also see the `verified as a null` message and I haven't received a confirmation email.

![image](https://github.com/community/community/assets/5190473/481699ee-8917-49fa-8323-3ae4add6523f)
![image](https://github.com/community/community/assets/5190473/ac7a56b1-7b21-429a-9f13-17497cfff01c)

Also related is #103082 with the same issue. Is this a bug? Thanks for your help.

**Reply by samus-aran** [reply]
Hi @Grimthorr 👋 

As you haven't yet received your academic benefits that is why it states "null". We'll update the internal team to address this to provide a more clearer message.

Seeing your screenshot you have only been approved 3 days ago. Your educational benefits will arrive within 72 hours of this verification. **However, during busy periods like the start of a new semester, it may take a bit longer.** 

We usually ask applications to wait until **5 days** before reaching out to the team regarding to their academic benefit status. Reply if you still haven't receive your benefits after the 5th day 👍

**Reply by Grimthorr** [reply]
Thanks @samus-aran, I received the email and benefits this morning.

Agreed; it's the `null` state that's causing confusion as it looks like a bug. It also doesn't mention the 72 hours timeline on that page which might be useful rather than "soon".

**Comment by wangcr2467** [comment]
Hi @samus-aran 
I have the same issue. My student status has been verified, but I have not received any benefits or email.
Thank you very much for taking the time to read my request, and I sincerely hope that you can help me.
![image](https://github.com/community/community/assets/61405252/49061e28-1b2f-43dd-acae-86fdf83f60ce)

**Reply by samus-aran** [reply]
Hi @wangcr2467 👋 

Similar to my other reply to @Grimthorr.

Seeing your screenshot you have only been approved 3 days ago. Your educational benefits will arrive within 72 hours of this verification. **However, during busy periods like the start of a new semester, it may take a bit longer.** 

We usually ask applications to wait until **5 days** before reaching out to the team regarding to their academic benefit status. Reply if you still haven't received your benefits after the 5th day 👍

**Comment by adam-osusky** [comment]
Hi @samus-aran, I have similar issue as @Nis-nischith. I see the same message:

![image](https://github.com/community/community/assets/65285737/319de0c3-f8cd-4b93-a4a0-341621a19606)

But my request was approved one month ago.

![image](https://github.com/community/community/assets/65285737/8d040064-ecad-44dd-be1b-67c519f5caa5)

Should I just wait or is there some problem? Thank you

**Reply by swiftalker** [reply]
is your problem has been solved now?

**Reply by adam-osusky** [reply]
No, I still do not have them.

**Comment by sudoplox** [comment]
<img width="703" alt="image" src="https://github.com/community/community/assets/12641646/1490a462-26cc-4b91-83bd-39497e7f7704">

facing a similar issue, unable to renew my license. it got expired on 8th april 2023, and now i cant click the Continue button as it is blocked. @samus-aran
